lib/Conversion/TTIRToTTMetal/TTIRToTTMetal.cpp

```diff
@@ -731,6 +733,13 @@ class TTIRToTTMetalDispatchRewriter : public OpRewritePattern<ttir::GenericOp> {
 
     // For all unary ops first copy tile from input CB at inCBTileIndex to DST
     // register at dstTileIndex.
```

Comment:
Move this comment below, above `CopyTileOp` creation.

Add missing comments that existed before above `TileRegsAcquireOp`, `TileRegsCommitOp` and `TileRegsWaitOp`.